### PR TITLE
OP-1035 - remove administrative flag

### PR DESCRIPTION
**Pull request by goto-eof**
See OP-1035.

- [x] Add paragraph with descriptions
- [x] Description for Patient Consensus flag OP-1036
- [x] Description for Additional Service flag OP-1038
- [x] Update screenshots

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient consensus
+Consent from a patient is needed regardless of the procedure, whether it's a physical examination or something else. The principle of consent is an important part of medical ethics and international human rights law.
+ In Open Hospital there are three types of Patient consent:
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
 In Open Hospital there are two types of Patient consent:
```

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
...
+- Consent to treatment: consent to treatment means a person must give permission before they receive any type of medical treatment, test or examination. This field is mandatory.
+- Consent to additional services - to introduce a flag in the patient registration form to record the patient consensus for additional services (optional, for the Portal)
+This consent should also be de-selectable, which implies the opt-out from the additional services (performed by the Patient Portal Connector)
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
This consent should also be de-selectable, which implies the opt-out from the additional services (performed by the Patient Portal Connector).
```

File: doc_user/UserManual.adoc [hunk 3]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient consensus
+Consent from a patient is needed regardless of the procedure, whether it's a physical examination or something else. The principle of consent is an important part of medical ethics and international human rights law.
+ In Open Hospital there are three types of Patient consent:
+- Consent to treatment: consent to treatment means a person must give permission before they receive any type of medical treatment, test or examination. This field is mandatory.
+- Consent to additional services - to introduce a flag in the patient registration form to record the patient consensus for additional services (optional, for the Portal)
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
- Consent to additional services - to introduce a flag in the patient registration form to record the patient consensus for additional services (optional, for the Portal).
```

File: doc_user/UserManual.adoc [hunk 4]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient consensus
+Consent from a patient is needed regardless of the procedure, whether it's a physical examination or something else. The principle of consent is an important part of medical ethics and international human rights law.
+ In Open Hospital there are two types of Patient consent:
+- Consent to treatment: consent to treatment means a person must give permission before they receive any type of medical treatment, test or examination. This field is mandatory.
+- Consent to additional services - to introduce a flag in the patient registration form to record the patient consensus for additional services (optional, for the Portal).
+This consent should also be de-selectable, which implies the opt-out from the additional services (performed by the Patient Portal Connector).
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: Why has been put like it was a requirement!? Please accept this suggestion

```suggestion
- Consent to additional services - consent to additional services means that the person consents to the use of the registered data for additional services, such as communications, mails, calls, follow ups reminders, digital services, etc. This field is NOT mandatory, but one hospital may decide to introduce it in its paper forms.
```

**Review by mwithi — CHANGES_REQUESTED** [review]
Please update the second consent as suggested, thanks!

**Comment by mwithi** [comment]
close because there was already https://github.com/informatici/openhospital-doc/pull/223

**Review by mwithi — APPROVED** [review]

**Comment by mwithi** [comment]
Only screenshot missing and we are done! Thanks!

**Comment by mwithi** [comment]
Superseded by https://github.com/informatici/openhospital-doc/pull/227

**Pull request closed without merging**